docs/design-proposals/cloud-edge-communication/cloud-edge-communication-framework.md

```diff
@@ -11,18 +11,255 @@ Because the cloud and edge is located in the network boundary of different secur
 However, there may be many problems when VPN is used to directly connect the components/services of different security levels:
```

Comment:
"can be connected through the VPN layer" 

Is this the only other option? is there any review done that collects and compares "current" solutions?

Reply:
Use the vpn for  the data communication between cloud and edge is not the only option， Direct Connect or just use public ip over internet are other options. Vpn is a common choice for data communication between different site (small data center) .
I will add some descriptions here, thx